app/build.gradle

```diff
@@ -1,59 +1,39 @@
 buildscript {
     repositories {
-        maven { url 'https://maven.fabric.io/public' }
+        maven { url "https://maven.fabric.io/public" }
     }
 
     dependencies {
         //noinspection GradleDynamicVersion
-        classpath 'io.fabric.tools:gradle:1.+'
+        classpath 'io.fabric.tools:gradle:1.27.0'
     }
 }
 
-apply plugin: 'com.android.application'
-apply plugin: 'io.fabric'
-
 repositories {
-    maven { url 'https://maven.fabric.io/public' }
+    maven { url "https://maven.fabric.io/public" }
```

Comment:
I think it's fine to leave this (and the other `maven.fabric.io/public` repo) as single-quote strings. Double-quotes are generally used with string interpolation (like for the release numbers under the dependencies), and we don't really need those for URLs at all.